conan/tools/cmake/cmakedeps.py

```diff
@@ -358,7 +358,7 @@ class CMakeDeps(object):
     components_target_build_type_tpl = Template(textwrap.dedent("""\
         ########## MACROS ###########################################################################
         #############################################################################################
-        include(${{CMAKE_CURRENT_LIST_DIR}}/cmakedeps_macros.cmake)
+        include(${CMAKE_CURRENT_LIST_DIR}/cmakedeps_macros.cmake)
```

Comment:
there is no test for this, right? components and CMakeDeps is barely tested

Reply:
Not really, I think it was partially discovered by some changes I did in the test, but not a explicit one. The thing is that we will move (copying first) all existing tests in cmake_find_package_multi to CMakeDeps, so all this will be more than covered, so it is probably not deserving its own new test for this.